linode/manager
# M3-5618: Update story for colors

## Description

- Clean up the colors used throughout the app and remove `cmr` from the naming convention
- Apply colors to the component itself if they're only used once instead of cluttering the theme colors
- Reorganize the layout to prevent the text overlapping

## How to test

- Run `yarn storybook`
- Go to `http://localhost:6006/?path=/docs/core-color-palette--color-palette`
- Make sure none of the color changes regressed on the app in light and dark mode
